src/main/java/org/opentripplanner/netex/mapping/StopMapper.java

```diff
@@ -47,7 +50,9 @@ Stop mapQuayToStop(
       idFactory.createId(quay.getId()),
       parentStation.getName(),
       quay.getPublicCode(),
-      quay.getDescription() != null ? quay.getDescription().getValue() : null,
+      quay.getDescription() != null
```

Comment:
You can use `NonLocalizedString#ofNullable` for these

Reply:
StopPlace’s description may be null, at least in tests.  If we only use `NonLocalizedString#OfNullable` here, it will cause nullpointer, since stopPlace.getDescription() returns null.